get.sh

```diff
@@ -286,7 +286,11 @@ getBinaryOpenjdk()
 				# Artifactory cannot handle duplicate slashes (//) in URL. Remove // except after http:// or https://
 				download_url_base=$(echo "$download_url_base" | sed -r 's|([^:])/+|\1/|g')
 				echo "artifactory URL: ${download_url_base}"
-				download_api_url_base=(${download_url_base//\/ui\/native\//\/artifactory\/api\/storage\/})
+				if [[ $download_url_base = *"ui/native"* ]]; then
+					download_api_url_base=(${download_url_base//\/ui\/native\//\/artifactory\/api\/storage\/})
+				else
+					download_api_url_base=(${download_url_base//\/artifactory\//\/artifactory\/api\/storage\/})
+				fi
```

Comment:
If the URL contains `artifactory/api/storage`, this will become `artifactory/api/storage/api/storage`.